ci-operator/step-registry/upi/install/azure/upi-install-azure-commands.sh

```diff
@@ -0,0 +1,309 @@
+export WORKER_IGNITION
+az deployment group create -g $RESOURCE_GROUP \
+  --template-file "06_workers.json" \
+  --parameters workerIgnition="$WORKER_IGNITION" \
+  --parameters sshKeyData="$SSH_PUB_KEY" \
+  --parameters baseName="$INFRA_ID"
+
+popd
+echo "Waiting for bootstrap to complete"
+openshift-install --dir=${ARTIFACT_DIR}/installer wait-for bootstrap-complete &
+wait "$!" || gather_bootstrap_and_fail
+
+echo "Bootstrap complete, destroying bootstrap resources"
+az network nsg rule delete -g $RESOURCE_GROUP --nsg-name ${INFRA_ID}-nsg --name bootstrap_ssh_in
+az vm stop -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap
+az vm deallocate -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap
+az vm delete -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap --yes
+az disk delete -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap_OSDisk --no-wait --yes
+az network nic delete -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap-nic --no-wait
+az storage blob delete --account-key $ACCOUNT_KEY --account-name $ACCOUNT_NAME --container-name files --name bootstrap.ign
+az network public-ip delete -g $RESOURCE_GROUP --name ${INFRA_ID}-bootstrap-ssh-pip
+
+echo "Adding ingress DNS records"
+
+export KUBECONFIG=${ARTIFACT_DIR}/installer/auth/kubeconfig
+
+set +e
+public_ip_router="<pending>"
+while [[ $public_ip_router =~ "pending" ]]
+do
+  public_ip_router=$(oc -n openshift-ingress get service router-default --no-headers | awk '{print $4}')
+  echo $public_ip_router
+done
+set -e
+
+az network dns record-set a add-record -g $BASE_DOMAIN_RESOURCE_GROUP -z ${BASE_DOMAIN} -n *.apps.${CLUSTER_NAME} -a $public_ip_router --ttl 300
+
+az network private-dns record-set a create -g $RESOURCE_GROUP -z ${CLUSTER_NAME}.${BASE_DOMAIN} -n *.apps --ttl 300
+az network private-dns record-set a add-record -g $RESOURCE_GROUP -z ${CLUSTER_NAME}.${BASE_DOMAIN} -n *.apps -a $public_ip_router
+
+function approve_csrs() {
+  oc version --client
+  while true; do
+    if [[ ! -f /tmp/install-complete ]]; then
+      # even if oc get csr fails continue
+      oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve || true
+      sleep 15 & wait
+      continue
+    else
+      break
+    fi
+  done
+}
+
+echo "Approving pending CSRs"
+export KUBECONFIG=${ARTIFACT_DIR}/installer/auth/kubeconfig
+approve_csrs &
+
+set +x
+echo "Completing UPI setup"
+openshift-install --dir=${ARTIFACT_DIR}/installer wait-for install-complete 2>&1 | grep --line-buffered -v password &
+wait "$!"
+
+date "+%F %X" > "${SHARED_DIR}/CLUSTER_INSTALL_END_TIME"
+# Password for the cluster gets leaked in the installer logs and hence removing them.
+sed -i 's/password: .*/password: REDACTED"/g' ${ARTIFACT_DIR}/installer/.openshift_install.log
+cp "${ARTIFACT_DIR}/installer/metadata.json" "${SHARED_DIR}"
+cp "${ARTIFACT_DIR}/installer/auth/kubeconfig" "${SHARED_DIR}"
+ls ${SHARED_DIR}
```

Comment:
Let's remove this. I assume that this was here for debugging purposes.
/lgtm cancel